rotkehlchen/api/v1/schemas.py

```diff
@@ -945,12 +946,37 @@ def make_history_base_entry(
         ) -> dict[str, Any]:
             return {'event': EthWithdrawalEvent(**data)}
 
+    class CreateAssetMovementEventSchema(BaseSchema):
+        event_type = SerializableEnumField(
+            enum_class=HistoryEventType,
+            exclude_types=[
+                event_type for event_type in HistoryEventType
+                if event_type not in (HistoryEventType.DEPOSIT, HistoryEventType.WITHDRAWAL)
+            ],
+            required=True,
+        )
+        asset = AssetField(required=True, expected_type=Asset, form_with_incomplete_data=True)
+        event_identifier = fields.String(required=False, load_default=None)
+        unique_id = fields.String(required=False, load_default=None)
```

Comment:
you have extra_data and this, while this is supposed to be part of extra data. This seems wrong.

Reply:
`unique_id` is only used to create the event_identifier - it doesn't actually get put in `extra_data` automatically. Of course sometimes the value used for `unique_id` also gets added to `extra_data`, for instance if its an onchain tx hash or something, but I think there should be be the option of specifying a unique_id here.